src/components/are-you-sure-dialog.tsx

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useMemo, useState } from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
If I'm not mistaken, all the changes in this file can be reverted because the new dialog isn't being used anymore.